modules/dashboard/php/endpoints/charthelper.class.inc

```diff
@@ -0,0 +1,308 @@
+<?php declare(strict_types=1);
+/**
+ * POST request methods.
```

Comment:
Can you remove this line and add a brief description for the file?

Reply:
@johnsaigle you want me to remove the `<?php declare(strict_types=1);` line? The description is on line 5 but I'll add more detail like building charts.

Reply:
No I meant removing "* POST request methods." because that's not a lot of information and also doesn't describe what the file is for.